.github/workflows/ci.yml

```diff
@@ -84,6 +84,7 @@ jobs:
       - name: build
         uses: ./.github/actions/build
         with:
+          SUDO: ""
```

Comment:
The `build` action does not take a `SUDO` input so we don't need to specify it here